web/src/components/hooks/useFeatures.ts

```diff
@@ -17,11 +17,15 @@ const noFeatures: features = {
   incidents: false,
 };
 
+// monitoring-console-plugin proxy via. cluster observability operator
+// https://github.com/rhobs/observability-operator/blob/28ac1ba9e179d25cae60e8223bcf61816e23a311/pkg/controllers/uiplugin/monitoring.go#L44
+const MCP_PROXY_PATH = '/api/proxy/plugin/monitoring-console-plugin/backend';
+
 export const useFeatures = () => {
   const [features, setFeatures] = React.useState<features>(noFeatures);
 
   const [response, loadError, loading] = useURLPoll<featuresResponse>(
```

Comment:
Do we need to polll this? or would it be enough to request it on mount?

Reply:
Updated this to only request on mount. There are now two hooks: useFeatures() and useFeaturesPolling().